otlp/attr_to_string.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2020 Splunk, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package otlp
+
+import (
+	"encoding/json"
+
+	otlpcommon "github.com/signalfx/sapm-proto/gen/otlp/common/v1"
+)
+
+// attributeMapToString converts an OTLP AttributeMap to a standard go map
```

Comment:
Comment does not match the functionality.

Reply:
Done.